mdbook/src/15-interrupts/the-mb2-speaker.md

```diff
@@ -0,0 +1,31 @@
+# The MB2 Speaker
+
+Your MB2 has a built-in speaker — the large black square device labeled "SPEAKER" in the middle of
+the back of the board.
+
+The speaker works by moving air in response to a GPIO pin: when the speaker pin is high (3.3V) a
+diaphragm inside — the "speaker cone" — is pushed all the way out; when the speaker pin is low (GND)
+it is pulled all the way back in. As air is pushed out and sucked back in, it flows in and out of
+the tiny rectangular hole — the "speaker port" — on the side of the device.  Do this fast enough,
+and the pressure changes will make a sound.
+
+<img class="white_bg" height="350" title="Speaker" src="../assets/speaker.svg" />
```

Comment:
In a dark theme, the image is barely visible as there is no background color. Currently, mdbook doesnt seem to support the `#only-dark`, `#only-light` tags for images to create two images that are shown in either light or dark mode. 
Looking at the Rust book as an example, the solution seems for the moment to give images solid white backgrounds. Attached an svg, if you like to use it, where I just took the existing figure and added a solid white background.

![Image](https://github.com/user-attachments/assets/8ac581c5-121d-4426-9dca-3be1be493a85)